nilearn/image/tests/test_resampling.py

```diff
@@ -404,34 +435,47 @@ def test_raises_bbox_error_if_data_outside_box():
     img = Nifti1Image(data, affine)
 
     # some axis flipping affines
-    axis_flips = np.array(list(map(np.diag,
-                              [[-1, 1, 1, 1],
-                               [1, -1, 1, 1],
-                               [1, 1, -1, 1],
-                               [-1, -1, 1, 1],
-                               [-1, 1, -1, 1],
-                               [1, -1, -1, 1]])))
+    axis_flips = np.array(
+        list(
+            map(
+                np.diag,
+                [
+                    [-1, 1, 1, 1],
+                    [1, -1, 1, 1],
+                    [1, 1, -1, 1],
+                    [-1, -1, 1, 1],
+                    [-1, 1, -1, 1],
+                    [1, -1, -1, 1],
+                ],
+            )
+        )
+    )
```

Comment:
Maybe better to set the list of list of integers to a variable and then call the functions on it

Reply:
good point: extracted it into a variable